README.md

```diff
@@ -70,7 +71,6 @@ Name|Category|Description|Last commit
 [Tigmint](https://github.com/bcgsc/tigmint)|assembly|Correct misassemblies using Linked Reads|![GitHub last commit](https://img.shields.io/github/last-commit/bcgsc/tigmint?label=%20)
 [TitanCNA_10x](https://github.com/GavinHaLab/TitanCNA_10X_snakemake)|pipeline,structural variants,cancer|Snakemake workflow for 10X Genomics WGS analysis using TitanCNA|![GitHub last commit](https://img.shields.io/github/last-commit/GavinHaLab/TitanCNA_10X_snakemake?label=%20)
 [Topsorter](https://github.com/hanfang/Topsorter)|structural variants, qc|Graphic assement of structural variants|![GitHub last commit](https://img.shields.io/github/last-commit/hanfang/Topsorter?label=%20)
-[VISOR](https://github.com/davidebolo1993/VISOR)|simulation|VarIant SimulatOR for short, long and linked reads|![GitHub last commit](https://img.shields.io/github/last-commit/davidebolo1993/VISOR?label=%20)
```

Comment:
Why remove `VISOR` from this list? It is published as a tool for linked-read simulation tool and is still portrayed as such on their [Github page](https://github.com/davidebolo1993/VISOR).